tasks/precheck.sh

```diff
@@ -11,6 +11,9 @@ elif grep -qi redhat /etc/os-release && [[ "$(cat /proc/sys/crypto/fips_enabled)
   osfamily="redhatfips"
 else
   osfamily="el"
+  if grep -qi amazon /etc/os-release && grep -qi "VERSION_ID=\"2\"" /etc/os-release; then
```

Comment:
I think we can simplify it a bit:
```suggestion
  if grep -qi amazon /etc/os-release && grep -qi 'VERSION_ID="2"' /etc/os-release; then
```